lib/browser/growl.js

```diff
@@ -63,23 +63,29 @@ exports.notify = function(runner) {
  */
 function isPermitted() {
   var permitted = {
-    granted: Promise.resolve(true),
-    denied: Promise.resolve(false),
-    default: (function ask() {
-      Notification.requestPermission().then(function(permission) {
-        return Promise.resolve(permission === 'granted');
+    granted: function allow() {
+      return Promise.resolve(true);
+    },
+    denied: function deny() {
+      return Promise.resolve(false);
+    },
+    default: function ask() {
+      return new Promise(function(resolve, reject) {
```

Comment:
the `new Promise` seems unneeded.  if you're going to eat a rejection, then just use `.catch()`

```js
default: function ask() {
  return Notification.requestPermission()
    .then(function(permission) {
      return permission === 'granted';
    });
```